src/frontend/src/session/cursor_manager.rs

```diff
@@ -384,6 +405,10 @@ impl SubscriptionCursor {
                             )?)));
                         }
                     } else {
+                        self.cursor_metrics
+                            .subscription_cursor_query_duration
+                            .with_label_values(&[&self.cursor_name])
+                            .observe(init_query_timer.elapsed().as_secs_f64());
```

Comment:
We use `timeseries_ms` in the grafana dashboard so we should use `as_millis` here. Also remember to change the histogram bucket accordingly as well.